BodoSQL/bodosql/context.py

```diff
@@ -1104,6 +1104,9 @@ def sql(self, sql, params_dict=None, dynamic_params_list=None, **jit_options):
             # are not visible to cloudpickle as used by the function. See:
             # test_json_fns.py::test_object_construct_keep_null[no_nested-no_null-with_case]
             if isinstance(dispatcher, SubmitDispatcher):
+                # __builtins__ which is added to glbls by exec causes issues in
+                # Jupyter/IPython pickling.
+                glbls.pop("__builtins__", None)
```

Comment:
Don't we need this to use builtin functions like len though?

Reply:
Builtin functions are the same on the worker and are available in regular globals. We don't need to pickle and send them.

Reply:
Makes sense, thanks